tests/rptest/tests/cluster_features_test.py

```diff
@@ -202,6 +203,49 @@ def test_explicit_activation(self):
         assert state['state'] == 'disabled'
         assert state['was_active'] == True
 
+    @cluster(num_nodes=3, log_allow_list=RESTART_LOG_ALLOW_LIST)
+    def test_explicit_activation(self):
+        """
+        Test uploading and retrieval of license
+        """
+        license = "eyJ2ZXJzaW9uIjogMCwgIm9yZyI6ICJyZWRwYW5kYSIsICJ0eXBlIjogMCwgImV4cGlyeS"\
+            "I6ICIyMTIyLTUtMzEifQ==."\
+            "SxZhWeVOuEnUNUjx068mhy8ftfkQHe0rRzdEVScIpRTBXd4Qe3G7cRNlklEqyiNwrHpnO9"\
+            "d0JoPbuatgkqDhvLnNo/rIL403jM6llksPbuVhK3rEs5nl3lBSbsCQf4AO/"\
+            "qmNlkxGu5o77Gy6blV3Vg78mL4JVtP6fFKUDbH8F4p32pDI8DxRkYtdxVdVvnDty6wgxe3"\
+            "xXz5HGMn1VWIr+X6jiJxKlA6JdjtH5/GIAxRTc5fAFf63I3Jo5BcGB5LAc/"\
+            "i+BI6UHmeOVFXlWrtEe5tx2yOtP/Fc6PNyqzgv8HWQLQdi6VURIIKDJ9ZfWjbjX/"\
+            "Bxa+bPvopAEXUWih6Ycw=="
+        license_contents = {
+            'expires': datetime.date(2122, 5, 31),
+            'version': 0,
+            'org': 'redpanda',
+            'type': 'free_trial'
+        }
+
+        assert self.admin.put_license(license).status_code == 200
```

Comment:
Either here or in the unit test, we should have the contrary cases:
- twiddle one byte in the signature and check it is rejected
- twiddle one attribute in the body and check it is rejected.
- empty body is rejected